apps/web/pages/video/[uid].tsx

```diff
@@ -36,19 +39,20 @@ export default function JoinCall(props: JoinCallPageProps) {
         width: "100%",
         height: "100%",
       },
+      url: meetingUrl,
+      ...(typeof meetingPassword === "string" && { token: meetingPassword }),
     });
-    callFrame.join({
-      url: props.booking.references[0].meetingUrl ?? "",
-      showLeaveButton: true,
-      ...(props.booking.references[0].meetingPassword
-        ? { token: props.booking.references[0].meetingPassword }
-        : null),
-    });
-  }, [props.booking?.references]);
+    callFrame.join();
+    return () => {
+      callFrame.destroy();
+    };
+  }, []);
+
+  const title = `${APP_NAME} Video`;
```

Comment:
this change fixed this warning.

`
Warning: A title element received an array with more than 1 element as children. In browsers title Elements can only have Text Nodes as children. If the children being rendered output more than a single text node in aggregate the browser will display markup and comments as text in the title and hydration will likely fail and fall back to client rendering
`
for more context : https://github.com/vercel/next.js/discussions/38256

Reply:
NIT I'm a proponent here for inline substitution, as it is unlikely the `title` var will be used anywhere else and is harder to read over inline.

```jsx
<title>{`${APP_NAME} Video`}</title>
```